proxy/state.go

```diff
@@ -60,7 +60,7 @@ func newProxyStateFromConfig(cfg *config.Config) (*proxyState, error) {
 	if err != nil {
 		return nil, err
 	}
-	state.authenticateDashboardURL = state.authenticateURL.ResolveReference(&url.URL{Path: dashboardPath})
+	state.authenticateDashboardURL = state.authenticateURL.ResolveReference(&url.URL{Path: "/.pomerium/"})
```

Comment:
It looks like this was changed intentionally in the original fix.  Unsure if that should be retained.

Reply:
Sorry I've lost track of these changes. I can't really review this for accuracy. I feel like this is the 7th time I've been asked for a code review on the same thing.

Are you saying we should use the variable name? My understanding is that the code in authenticate adds a trailing `/`, so our redirect URI fails HMAC validation because the url is changed. But these changes were made a while ago so maybe I'm misremembering.